doc/source/multivisit.rst

```diff
@@ -3,4 +3,535 @@
+  The projection cells are 4°x4° in size and are on rings spaced by 4° in declination.
+
+MVM processing uses these pre-defined projection cells after extending them to cover the entire sky
+with cells that are actually 4.2°x4.2° in size and are on rings spaced by 4° apart in declination.  This provides
+sufficient overlap to allow data from one projection cell to be merged with data from a neighboring cell if necessary.
+These cells, defined as the ProjectionCell object in the code and referred to as **ProjectionCell**, are indexed
+starting at 0 at δ = −90° (South Pole) and ending at 2643 at δ = +90° (North Pole).  As stated in the
+`PanSTARRS page <https://outerspace.stsci.edu/display/PANSTARRS/PS1+Sky+tessellation+patterns>`_:
+
+  The projection cell centers are located on lines of constant declination spaced 4° apart. At a given declination,
+  the pointing centers are equally spaced in right ascension around the sky, with the number of RA points changing to
+  account for the convergence of RA lines in the spherical sky. The pattern is defined to cover the entire sky from
+  δ = −90° to +90°... Within a given declination zone, the projection cells are centered at RA(n) = n Δα = n 360°/M where
+  M is the number of RA cells in the zone. Finally, the projection cells themselves are numbered consecutively
+  (ordered by increasing RA) starting at 0 at the south pole, 1–9 at δ = −86°, etc.
+
+All these definitions, including how finely each ProjectionCell should be divided to define the SkyCells, are encoded in a
+FITS table installed as part of the `drizzlepac` package:
+
+    drizzlepac/pars/allsky_cells.fits
+
+
+Defining each SkyCell
+----------------------
+Each ProjectionCell is split into a grid of 21 x 21 'sky cells' which serves as the most basic MVM product generated
+during MVM processing.  Sky cells, as defined as the SkyCell object in the code and referred to as **SkyCell**,
+are approximately 0.2° x 0.2° in size (~21500 x ~21500 pixels) and
+they have the same WCS as the ProjectionCell.  Each SkyCell gets identified by its position within the ProjectionCell
+as shown in this figure:
+
+.. figure:: images/SkyCell_numbering.png
+  :width: 90%
+  :alt: Numbering convention for SkyCells within a Projection Cell.
+
+  Numbering convention for SkyCells within a Projection Cell used for naming the SkyCell.
+
+The numbering of the SkyCells within a ProjectionCell starts in the lower left corner at (1,1) corresponding to the cell
+that with the lowest declination and largest RA since the SkyCell is oriented so that the Y axis follows a line of RA
+pointing towards the North Pole.
+This indexing provides a way to identify uniquely any position on the sky that can be used as the basis for a unique filename for
+all products generated from the exposures that overlap each SkyCell.  Mosaics generated for each SkyCell uses this
```

Comment:
Change:
sky that can be used as the ==> sky and can be used as the

Reply:
Fixed in 3cd7f7e.